### PR TITLE
Android and cross things

### PR DESCRIPTION
**Pull request by lasote**
Review again to see the new section

### CODE REVIEW

File: integrations/android_studio.rst [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+
+Use can use Conan to :ref:`cross-build your libraries for Android<cross_building_android>` in different architectures.
+If you are using Android Studio for your Android application development, you can integrate it conan to automate the
+library building for the different architectures that you want to support in your project.
+
+Here is an example of how to integrate the ``libpng`` conan package library in an Android application, but any library
+that can be cross-compiled to Android could be used using the same procedure.
+
+We are going to use the "Hello World" application as an example:
```
Comment (memsharded) [inline integrations/android_studio.rst]: Not clear, you are not doing a Hello World, you are doing an app with libpng, zlib.

File: howtos/cross_building.rst [hunk 2]
```diff
@@ -103,7 +103,7 @@ From Windows to Raspberry PI
         compiler.version: 4.6
         compiler.libcxx: libstdc++
         build_type: Debug
-        arch: armv6
+        arch: armv7hf
```
Comment (memsharded) [inline howtos/cross_building.rst]: why did you change it?

File: howtos/cross_building.rst [hunk 3]
```diff
@@ -143,3 +143,155 @@ If you have your project building automated with conan you can then just call ``
...
+-------
+
+
+Cross bulding a library for Android is very similar to the previous examples, except the complexity of managing different
+architectures (armeabi, armeabi-v7a, x86, arm64-v8a) and the Android api levels.
+
+Creating a toolchain for the specific settings can be a tedious task. The Android NDK offers a mechanism to create a
+`standalone toolchain <https://developer.android.com/ndk/guides/standalone_toolchain.html?hl=es>`_ for different settings.
+
+We created a recipe ``android-toolchain/r13b@lasote/testing`` to be used as a :ref:`build requirement<build_requires>`.
```
Comment (memsharded) [inline howtos/cross_building.rst]: Not sure if the "bare" example of the toolachain without the "build_requires" should be explained somewhere too.

File: howtos/cross_building.rst (hunk 2 above)
Comment (lasote) [inline howtos/cross_building.rst]: The RPI 1 and 2 architecure is armv7hf, not armv6

File: howtos/cross_building.rst (hunk 3 above)
Comment (lasote) [inline howtos/cross_building.rst]: You mean my conan toolchain package or directly the NDK toolchain?

Comment (lasote) [inline howtos/cross_building.rst]: Added new section

File: howtos/cross_building.rst [hunk 4]
```diff
@@ -295,3 +510,27 @@ use environment variables:
     - See `CMake cross building wiki <http://www.vtk.org/Wiki/CMake_Cross_Compiling>`_ to know more about cross building with CMake.
 
 
+Useful conans.tools for cross building
+--------------------------------------
+
+.. code-block:: python
```
Comment (memsharded) [inline howtos/cross_building.rst]: Maybe this code-block a bit naive, only to explain the imports
